components/cargo.py

```diff
@@ -65,10 +70,10 @@ def setup(self) -> None:
 
     def execute(self) -> None:
         self.intake_motor.set(ctre.ControlMode.PercentOutput, self.intake_motor_output)
-
-        self.pid_controller.setReference(self.setpoint)
-        output = self.pid_controller.update()
-        self.arm_motor.set(output)
+        self.pid_controller.setReference(self.setpoint, rev.ControlType.kSmartMotion)
+        wpilib.SmartDashboard.putNumber(
+            "cargo_output", self.arm_motor.getAppliedOutput()
+        )
```

Comment:
Don't put to the SmartDashboard class, this is inefficient.

If you want to regularly get feedback, use the magicbot feedback instead. But it seems like this isn't very useful anyway.